native_client/java/libdeepspeech/src/main/java/org/mozilla/deepspeech/libdeepspeech/DeepSpeechModel.java

```diff
@@ -1,12 +1,17 @@
-package deepspeech.mozilla.org.deepspeech;
+package org.mozilla.deepspeech.libdeepspeech;
 
-public class Model {
+public class DeepSpeechModel {
+
+    static {
+        System.loadLibrary("deepspeech-jni");
+        System.loadLibrary("deepspeech");
+    }
 
     // FIXME: We should have something better than those SWIGTYPE_*
     SWIGTYPE_p_p_ModelState _mspp;
     SWIGTYPE_p_ModelState   _msp;
 
-    public Model(String modelPath, int n_cep, int n_context, String alphabetPath, int beam_width) {
+    public DeepSpeechModel(String modelPath, int n_cep, int n_context, String alphabetPath, int beam_width) {
```

Comment:
As of v0.4.1, the fields `alphabetPath` and `beam_width` have been renamed,
```suggestion
    public DeepSpeechModel(String modelPath, int n_cep, int n_context, String lmAlpha, int lmBeta) {
```

Reply:
They did not change, those were the decoder parameters.